src/main/java/parkSystem/park/common/dto/response/CommonResponse.java

```diff
@@ -0,0 +1,11 @@
+package parkSystem.park.common.dto.response;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public class CommonResponse {
```

Comment:
Dto를 사용할때 record를 사용하는것도 괜찮은 방법 같슽니다 
```suggestion
public record CommonResponse(String code, String messgae) {
}